omod/src/main/resources/config.xml

```diff
@@ -93,6 +93,10 @@
 		<name>Export Patient Data</name>
 		<description>Ability to export patient data in FHIR format</description>
 	</privilege>
+	<privilege>
+		<name>Export Non Anonymised Patient Data</name>
+		<description>Ability to export non anonymised patient data in FHIR format</description>
```

Comment:
Ability to bulk-export patient data that has NOT been anonymised, in FHIR JSON format.